package.json

```diff
@@ -92,7 +92,7 @@
     "@types/uuid": "^8.3.4",
     "@typescript-eslint/eslint-plugin": "^5.19.0",
     "@typescript-eslint/parser": "^5.0.0",
-    "cypress": "13",
+    "cypress": "13.2.0",
```

Comment:
is it normal for us to specify specific versions of Cypress rather than semver versions?

Reply:
no, this should be semver'd. i'll adjust